### PR TITLE
release runtime v24

### PR DESCRIPTION
**Pull request by ianhe8x**

### CODE REVIEW

**Comment by xlc** [comment]
Not sure why CI build WASM failed. Does it work for you locally?

File: runtime/src/lib.rs [hunk 1]
```diff
@@ -72,8 +72,8 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("cennznet"),
 	impl_name: create_runtime_str!("centrality-cennznet"),
 	authoring_version: 1,
-	spec_version: 24,
-	impl_version: 24,
+	spec_version: 25,
```
Comment (xlc) [inline runtime/src/lib.rs]: keep those 24. bump them after new release is tagged

**Comment by ianhe8x** [comment]
> Not sure why CI build WASM failed. Does it work for you locally?
Yes, it works on my local

**Comment by ianhe8x** [comment]
this is my local rust version
```
Default host: x86_64-apple-darwin

installed toolchains
--------------------

stable-x86_64-apple-darwin
nightly-x86_64-apple-darwin (default)

installed targets for active toolchain
--------------------------------------

asmjs-unknown-emscripten
wasm32-unknown-unknown
wasm32-wasi
x86_64-apple-darwin

active toolchain
----------------

nightly-x86_64-apple-darwin (default)
rustc 1.36.0-nightly (a9ec99f42 2019-05-13)
```

**Pull request merged**